### PR TITLE
Updated Phenotype modal

### LINKED ISSUE #311: Updates to phenotype display
We made a few changes to the phenotype intake app:

```
"age_at_last_evaluation" -> "age_at_enroll" (Refers to age at PediSeq enrollment)
"age_at_last_evaluation_units" -> "age_at_enroll_units"
"age_at_initial_evaluation" -> "age_at_initial_dx" 
"age_at_initial_evaluation_units"-> "age_at_initial_dx_units"

"ethnicity" -> "ethnicity paternal" 
                     "ethnicity_maternal"
```

Two fields were added:

```
 "race_paternal"
 "race_maternal"
```

Label changes:

```
The label for Consanguinity should be changed to "Parental Consanguinity"
The label for Chief Complaint should be changed to "Chief Complaint/Reason for Visit"
```

The new endpoint is now live in development.

### PR DESCRIPTION
**Pull request by hassanNS**
Fix #311 
small changes
Signed-off-by: Sheik Hassan solergiga@yahoo.com

### CODE REVIEW

**Comment by hassanNS** [comment]
screen shot 2014-06-04 at 2 53 21 pm

updated

**Comment by naegelyd** [comment]
Since this is meant to address https://github.com/cbmi/varify/issues/311, do you need to change the `Consanguinity` label to `Parental Consanguinity`?

File: varify/static/templates/modals/phenotypes.html [hunk 1]
```diff
@@ -19,13 +19,19 @@ <h3 data-target=header-label></h3>
 
             <div class='row-fluid horizontal-list-display' data-target=personal-info>
                 <div class=span8><h6>Sex:</h6><%= data.sex %></div>
-                <div class=span8><h6>Ethnicity:</h6><%= data.ethnicity %></div>
-                <div class=span8><h6>Chief Complaint:</h6><%= data.chief_complaint %></div>
+                <div class=span8><h6>Ethnicity (Paternal):</h6><%= data.ethnicity_paternal %></div>
```
Comment (naegelyd) [inline varify/static/templates/modals/phenotypes.html:22]: no underscores, make this change throughout this whole file, use `ethnicity-paternal` instead

Comment (hassanNS) [inline varify/static/templates/modals/phenotypes.html:22]: but isn't that how it's represented in the json?

Comment (naegelyd) [inline varify/static/templates/modals/phenotypes.html:22]: Yea, you are right, I misread this as `data-target`. This is fine.

**Pull request merged**